src/tercerProblema.js

```diff
@@ -0,0 +1,41 @@
+//Reto: hacer uso de PROMESAS para evitar Callback Hell
+//Versión trabajada apartir del segundo problema
+
+var API = 'https://rickandmortyapi.com/api/character/';
+var xhttp = new XMLHttpRequest();
+
+const fetchData = url_api => {
+  return new Promise( (resolve, reject) => {//Se define la promesa
+    xhttp.onreadystatechange = event => {
+      if (xhttp.readyState === 4) {
+        if (xhttp.status === 200) {
+          resolve(JSON.parse(xhttp.responseText));//Se hace el llamado a resolve
+        }
+        else{
+          reject(url_api);//Se llama a reect en caso de error
```

Comment:
Te recomiendo ser mas claros en el reject, esto te permite identificar el error mas fácil. 

```suggestion
          reject(new Error("Algo salió mal."));
```